.github/workflows/markdown-lint.yml

```
@@ -31,26 +31,42 @@
           yamllint --version || { printf "%s\n" "${ERR_MSG_LINE}"; exit 1; }
 
       - name: Lint Workflow YAML
+        if: ${{ success() }}
         run: |
```

actionlint (1.7.4)

shellcheck reported issue in this script: SC1001:info:3:17: This \- will be a regular '-' in this context

(shellcheck)